charts/pega/README.md

```diff
@@ -939,22 +939,22 @@ Parameter   | Description   | Default value
 `service.tls.external_secret_name` | If you configured a secret in an external secrets operator, enter the secret name. For details, see [this section.](#optional-support-for-providing-credentialscertificates-using-external-secrets-operator) | `""` 
 `service.tls.keystore` | The keystore content for the tier. If you leave this value empty, the deployment uses the default keystore. | `""`
 `service.tls.keystorepassword` | The keystore password for the tier. If you leave this value empty, the deployment uses the default password for the default keystore. | `""`
-`service.tls.cacertificate` | The CA certificate for the tier. If you leave this value empty, the deployment uses the default CA certificate for the default keystore. Pass the certificateChainFile file if you are using certificateFile and certificateKeyFile | `""`
-`service.tls.certificateFile` | The content of the file that contains the server certificate. The format is PEM-encoded. This should be given if keystore and keystorepassword is not specified. | `""`
-`service.tls.certificateKeyFile` | The content of the file that contains the server private key. The format is PEM-encoded. This should be given if keystore and keystorepassword is not specified. | `""`
+`service.tls.cacertificate` | The CA certificate for the tier. If you leave this value empty, the deployment uses the default CA certificate for the default keystore. Pass the certificateChainFile file if you are using certificateFile and certificateKeyFile. | `""`
+`service.tls.certificateFile` | The content of the file that contains the server certificate, which you must provide if keystore and keystorepassword is not specified. The format of this file is PEM-encoded. | `""`
```

Comment:
make this the same as the next one>>
which you must provide if you do not provide a keystore and keystorepassword.

Reply:
done